### DISCUSSION TITLE
Custom filetr syntax ?

### DISCUSSION THREAD

**Discussion by gdnathan** [nvim-tree/nvim-tree.lua#1345] [Q&A]
What is the syntax for custom filters ? 
I have this:
```
    filters = {
        custom = {
            ".git",
            "node_modules",
            ".next"
        },
```

It does exclude the .git folder from the tree, that's what i want.
But it also exclude ANY files/folder STARTING with .git, like `.gitignore`, `.gitmodules`, .``github/``... How do i tell my filter to only exclude files that are exactly "`.git`" and not "`.git*`"

Many thanks

**Comment by kyazdani42** [comment]
these are based on `vim.regex`, add `.git$` if you want to filter out the .git folder :)
